### PR TITLE
Adds keyboard shortcuts for panning and zooming on a focused view

### PR DESCRIPTION
**Pull request by carolinebridge-oicr**
Adds keyboard shortcuts for:

Scroll with LeftArrow and RightArrow
Zoom with Shift + (UpArrow || DownArrow)

- put the event listener in the AppFocus mixin, wondering if that's the best place for it
- not sure if I like shift + arrow for zoom, I think ctrl/cmd is more intuitive but would like to hear thoughts

Waiting for the focus PR to be merged before this one into main 

Partially resolves https://github.com/GMOD/jbrowse-components/issues/927

### CODE REVIEW

**Comment by garrettjstevens** [comment]
I think ctrl+up/down is more intuitive for zoom than shift.

I noticed that if the side widget has a horizontal scroll and is in focus from a browser perspective (not from a JBrowse perspective) the left/right will also scroll the widget.

widget_scroll

File: plugins/linear-genome-view/src/LinearGenomeView/model.ts [hunk 1]
```diff
@@ -1516,6 +1516,31 @@ export function stateModelFactory(pluginManager: PluginManager) {
           : undefined
       },
     }))
+    .actions(self => ({
+      afterCreate() {
+        document.addEventListener('keydown', e => {
+          const session = getSession(self)
+          if (session.focusedView?.type === 'LinearGenomeView') {
```
Comment (cmdcolin) [inline plugins/linear-genome-view/src/LinearGenomeView/model.ts:1523]: maybe the code can change to be like "if the session.focusedView === self, then do these operations on self" rather calling session.focusedView.##actionName"

**Comment by cmdcolin** [comment]
i might agrew with ctrl+up/down because we have ctrl+scroll for zoom too

**Comment by carolinebridge-oicr** [comment]
Added ctrl/meta keys to the left and right operations --

- consistent with the other nav operations
- allows the user to still scroll up and down on widgets / in the main views container if they wish if they use the arrows without pressing ctrl/meta
- deeply nested preventDefault so if the user has the view in focus it doesn't override the default for every keystroke (e.g. cmd + r for refresh would be overridden if it was one level outside of where i placed it)

**Comment by cmdcolin** [comment]
I added a 'removeEventHandler' using an addDisposer. I think should be good to go. it is a little tricky, becuase I was pressing "ctrl+down" multiple times but it would not zoom out a lot. that is because it is actively doing an animation. it may be good to find a workaround for this...i think it would be good to 'quickly zoom out' a multiple rapid ctrl+down key presses.

**Pull request merged**